main.py

```diff
@@ -0,0 +1,21 @@
+
+N = 0
+while int(N) not in range(2, 7):
+    N = input("Enter a number between 2 to 6: ")
+deck = set()
+scores = list()
+for i in range(40):
+    deck.add(str(i+1))
+for i in range(int(N)):
+    card_text = deck.pop()
+    card_number = int(card_text)
+    scores.append(card_number)
+    curr_player_index = str(i+1)
+    curr_player_score = str(scores[i])
+    print(f"Player {curr_player_index} draw {curr_player_score}")
```

Comment:
f format take every non-string variable and activate str(..) of that. for example, the next code works. 
number = 10 
array = [2,3,4] 
print(f'I can print all types with f-string, for example, this is a number {number} and I can print even arrays {array}')